WIP 3.0 Classes/Priest.md

```diff
@@ -509,717 +568,122 @@ You learn the *calm emotions* spell and always have it prepared. In addition, yo
-*6th-level priest feature*<br/>
+*6th-level Shadow feature*<br/>
 <div style='margin-top:-4px;'></div>
 
 Your mind is more guarded to outside forces. You are immune to any magical effect that would read your thoughts or intentions, such as the *detect thoughts* spell, and Wisdom (Insight) check against you are made with disadvantage.  
 
-Additionally, if a creature attempts to charm or frighten you and you succeed on the saving throw, you can use your reaction to force the creature to make the same saving throw, becoming charmed or frightened of you until the end of your next turn if they fail instead.  
+Additionally, if a creature attempts to charm or frighten you and you succeed on the saving throw, you can use your reaction to force the creature to make the same saving throw against your priest spell save DC, becoming charmed or frightened of you until the end of your next turn if they fail instead.
 
 #### Horrific Visions
-*10th-level priest feature*<br/>
+*10th-level Shadow feature*<br/>
 <div style='margin-top:-4px;'></div>
 
-You learn how to invoke creature's deepest, darkest fears. When a creature starts their turn frightened or charmed by you, or while under the effect of one of your Shadow Calling spells, You can choose to have their movement halved and can only take either their action or their bonus action on that turn. 
+You learn how to invoke creature's deepest, darkest fears. When a creature starts their turn frightened or charmed by you, or while under the effect of one of your Shadow Calling spells, you can choose to have their movement halved and on its turn, it can use either an action or a bonus action, not both.
 
-####  Dark Absolution 
-*18th-level priest feature*<br/>
+#### Dark Absolution 
+*18th-level Shadow feature*<br/>
 <div style='margin-top:-4px;'></div>
 
-Your body becomes more one with shadowy magic. While you have Shadowform active, you can move through creatures and objects as though they were difficult terrain. If you end your turn inside a creature's space or an object, you take 5 force damage. 
+Your body becomes one with shadowy magic. While you have Shadowform active, you can move through creatures and objects as though they were difficult terrain. If you end your turn inside an object, you take 1d10 force damage. 
 
-In addition, you can manipulate even the strongest of minds. If you would inflict the frightened or charmed conditions on a creature who would be immune, the creature is treated as if it is frightened or charmed for the purpose of your priest abilities, and gains disadvantage on ability checks while you are in their line of sight. 
+In addition, you can manipulate even the strongest of minds. If you would try to inflict the frightened or charmed conditions on a creature who would be immune, the creature is treated as if it is frightened or charmed for the purpose of your priest abilities, and gains disadvantage on ability checks while you are in their line of sight. 
```

Comment:
Maybe this could be worded a bit differently.

> If you would try to inflict the frightened or charmed conditions on a creature who would be immune, 

It kind of reads like that only if you try to inflict the conditions the creature counts.

Maybe something along the line of 

> After trying to inflict the frightened or charmed conditions on a creature that is immune to them, the creature is treated as if it is frightened or charmed for the purpose of your priest abilities, and gains disadvantage on ability checks while you are in their line of sight. 

Also i think it needs clarification on when this effect ends.

Reply:
Updated text with this version, plus added "until the end of your next turn" to clarify duration.